server.js

```diff
@@ -1,27 +1,31 @@
 import express from "express";
-import cors from "cors";
 import mongoose from "mongoose";
+import cors from "cors";
+const thoughtRoutes = require("./routes/thoughtRouter");
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/thoughts";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
+// Defines the port the app will run on.
+const port = process.env.PORT || 9000;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
+// Adds middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
+app.use("/", thoughtRoutes)
+app.use((req, res, next) => {
```

Comment:
Had to read up on this middleware to see what it does. Have understood that it checks the connection to MongoDB before handling requests. Good job adding extra check ups to see that everything is working as expected!